main.py

```diff
@@ -1,15 +1,61 @@
+# Main
 from utils.console import print_markdown
+from utils.console import print_step
+from utils.console import print_substep
+from rich.console import Console
 import time
+import os
 from reddit.askreddit import get_askreddit_threads
 from video_creation.background import download_background, chop_background_video
 from video_creation.voices import save_text_to_mp3
 from video_creation.screenshot_downloader import download_screenshots_of_reddit_posts
 from video_creation.final_video import make_final_video
-
+from utils.loader import Loader
+from dotenv import load_dotenv
+console = Console()
 print_markdown(
     "### Thanks for using this tool! 😊 [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com). If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue."
 )
 
+"""
+
+Load .env file if exists. If it doesnt exist, print a warning and launch the setup wizard.
+If there is a .env file, check if the required variables are set. If not, print a warning and launch the setup wizard.
+
+"""
+
+client_id=os.getenv("REDDIT_CLIENT_ID")
+client_secret=os.getenv("REDDIT_CLIENT_SECRET")
+username=os.getenv("REDDIT_USERNAME")
+password=os.getenv("REDDIT_PASSWORD")
+
+console.log("[bold green]Checking environment variables...")
+time.sleep(1)
+
+if client_id == "" or client_secret == "" or username == "" or password == "":
```

Comment:
if not all(client_id, client_secret, username, password)

Reply:
ah, I'm so used to python2 where this is the only option!